cypress/integration/controlOperations.js

```diff
@@ -34,7 +34,7 @@ describe('Control operations', function() {
         });
     });
 
-    it('Change password -> Relogin -> Change password', () => {
+    /*it('Change password -> Relogin -> Change password', () => {
```

Comment:
Not sure you meant to commit this?

Reply:
I commented them out to not waste the recordings allowed for free plan with cypress :D I didn't want to run all tests while triggering the test travis builds :)